addon/templates/components/polaris-empty-search-result.hbs

```diff
@@ -1,14 +1,18 @@
 {{#polaris-stack
   alignment="center"
   vertical=true
+  as |stack|
 }}
   {{#if withIllustration}}
-    {{!-- TODO: replace this with polaris-image when implemented. --}}
-    <img
-      src="empty-search.svg"
-      alt="Empty search results"
-      class="Polaris-EmptySearchResult__Image"
-    >
+    {{#stack.item}}
+      {{!-- TODO: replace this with polaris-image when implemented. --}}
+      <img
+        src="empty-search.svg"
```

Comment:
Does this Just Work™️ if you use this in a real host app, or does it need the image file to be copied across?

Reply:
The image is added in this PR 😄

Reply:
When the main component is done we should (obviously) pull the branch into an app and make sure the path is correct but I _think_ this should be ok.

Reply:
Yeah agreed - there's an image in the main component too so we should check both 😉